### PR TITLE
Changed CN=quickstart references to greymatter.user

### PR DESCRIPTION
**Pull request by zakilur**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

This change is in response to issue #923 . Moving away from the quickstart certs required changing the CN to match that provided by the `greymatter-user-cert` secret.
<br/><br/>

2. What **changes to custom.yaml** are required?

None
<br/><br/>

3. Have you documented any additional setup steps or configurations options?

None
<br/><br/>

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Changes have been verified by deploying Fibonacci service to a local K3D cluster.
Confirmed the following as working:

- Fibonacci service, including mesh configs, was successfully deployed.
- Fibonacci card on the dashboard was displaying properly and was showing metrics from Prometheus
- SLOs were tested successfully; they displayed properly in the dashboard and were confirmed in Postgres
- Business Impact was tested successfully; Changes in business impact were reflected in Redis and the catalog entry

<br/><br/>

### CODE REVIEW

**Comment by chrisbsmith** [comment]
@zakilur The lint-charts test is failing. You probably need to update a Chart version. 
@moosecanswim would like you to review this also

**Comment by zakilur** [comment]
I'm looking to see what happened. I ran `helm lint` commands on the chart directories that changed and everything checks out. I'm looking to see if I can get more info on where the automated lint-check failed.

**Comment by zakilur** [comment]
It says it failed on linting `edge` and `secrets` but those weren't changed as part of this. I run a manual lint on both directories with no issues. The error in the lint checks seem to indicate some sort of issue pulling the Docker image for testing. I'm rerunning now with no changes and seeps to already be looking better.

**Comment by moosecanswim** [comment]
@zakilur @chrisbsmith I think you need to add the new certs into the repo too https://github.com/greymatter-io/helm-charts/tree/release-2.3/certs unless we want to host them elsewhere and have users download them manually.

**Comment by chrisbsmith** [comment]
> @zakilur @chrisbsmith I think you need to add the new certs into the repo too https://github.com/greymatter-io/helm-charts/tree/release-2.3/certs unless we want to host them elsewhere and have users download them manually.

@moosecanswim , we actually won't be hosting any more users certs in the repo.  PR #919 removed the need for this for "quickstart" and "demo" purposed.  We do need to update our documentation to reflect how users can pass in their own certs, but some of this will also be updated by issue #925

**Comment by moosecanswim** [comment]
> > @zakilur @chrisbsmith I think you need to add the new certs into the repo too https://github.com/greymatter-io/helm-charts/tree/release-2.3/certs unless we want to host them elsewhere and have users download them manually.
> 
> @moosecanswim , we actually won't be hosting any more users certs in the repo. PR #919 removed the need for this for "quickstart" and "demo" purposed. We do need to update our documentation to reflect how users can pass in their own certs, but some of this will also be updated by issue #925

Ah thats nifty! In this case can we remove the certs directory? We're generally pretty good about adding new stuff but not about removing fluff.

**Comment by chrisbsmith** [comment]
> Ah thats nifty! In this case can we remove the certs directory? We're generally pretty good about adding new stuff but not about removing fluff.

Yes, we need to remove the old certs.  @zakilur can you actually do that with this PR if you aren't already?  Just delete the whole `certs/` directory

**Comment by zakilur** [comment]
Certs directory has been removed

**Comment by zakilur** [comment]
So that broke the integration testing script.
```bash
=== RUN   TestCatalog
TestCatalog 2021-03-23T16:06:19Z client.go:33: Configuring kubectl using config file /home/runner/.kube/config with context 
2021/03/23 16:06:19 open ../certs/tls.crt: no such file or directory
FAIL	command-line-arguments	179.266s
FAIL
Error: Process completed with exit code 1.
```

**Comment by moosecanswim** [comment]
> So that broke the integration testing script.
> 
> ```shell
> === RUN   TestCatalog
> TestCatalog 2021-03-23T16:06:19Z client.go:33: Configuring kubectl using config file /home/runner/.kube/config with context 
> 2021/03/23 16:06:19 open ../certs/tls.crt: no such file or directory
> FAIL	command-line-arguments	179.266s
> FAIL
> Error: Process completed with exit code 1.
> ```

lol! law of unintended consequences

**Comment by zakilur** [comment]
> > So that broke the integration testing script.
> > ```shell
> > === RUN   TestCatalog
> > TestCatalog 2021-03-23T16:06:19Z client.go:33: Configuring kubectl using config file /home/runner/.kube/config with context 
> > 2021/03/23 16:06:19 open ../certs/tls.crt: no such file or directory
> > FAIL	command-line-arguments	179.266s
> > FAIL
> > Error: Process completed with exit code 1.
> > ```
> 
> lol! law of unintended consequences

I should be able to just have the extract function drop the certs in the base dir and look for them there.

**Comment by zakilur** [comment]
> > > So that broke the integration testing script.
> > > ```shell
> > > === RUN   TestCatalog
> > > TestCatalog 2021-03-23T16:06:19Z client.go:33: Configuring kubectl using config file /home/runner/.kube/config with context 
> > > 2021/03/23 16:06:19 open ../certs/tls.crt: no such file or directory
> > > FAIL	command-line-arguments	179.266s
> > > FAIL
> > > Error: Process completed with exit code 1.
> > > ```
> > 
> > 
> > lol! law of unintended consequences
> 
> I should be able to just have the extract function drop the certs in the base dir and look for them there.

Updated the script and gitignore. Waiting to see if the testing comes back clean.

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,9 @@ charts.txt
 *.log
 .DS_Store
 */Chart.lock
+tls.crt
```
Comment (chrisbsmith) [inline .gitignore:11]: I don't know that we _need_ these since the files are only written out in the test, but i'm ok with leaving it in here.

**Review by chrisbsmith — APPROVED** [review]
LGTM

File: .gitignore (hunk 1 above)
Comment (zakilur) [reply .gitignore:11]: Just in case a developer is testing locally and they end up pushing changes. It won't make a difference for the automated testing.

**Pull request merged**